ElementX/Sources/Screens/RoomScreen/ComposerToolbar/CompletionSuggestionService.swift

```diff
@@ -8,8 +8,8 @@
 import Combine
 import Foundation
 
-private enum SuggestionTriggerPattern: Character {
-    case at = "@"
+private enum SuggestionTriggerRegex: String {
+    case at = "@\\w+"
```

Comment:
No biggie if not, but any chance we could use Swift's `Regex` here instead of a string and an `NSRegularExpression`? That way it will be compile-time safe and you won't need to guard it.

```suggestion
    case at = /@\w+/
```

Reply:
It seems not to be working because the enum doesn't support the `Regex<Substring>` type, also the `NSRegularExpression` does not support the type either. maybe need to do more work on this change, what do you think?

Reply:
Oh I didn't spot that.

Just remove the `: String` for now and change the `case` to a `static let` - there's only trigger, we don't need the ability to iterate through them right now :)

Reply:
Ok! I'm already finished.